core/server/worker/src/main/java/alluxio/worker/netty/PipelineHandler.java

```diff
@@ -43,6 +45,8 @@ public PipelineHandler(WorkerProcess workerProcess) {
     mWorkerProcess = workerProcess;
     mFileTransferType = Configuration
         .getEnum(PropertyKey.WORKER_NETWORK_NETTY_FILE_TRANSFER_TYPE, FileTransferType.class);
+    mRequestManager = new AsyncCacheRequestManager(
+        NettyExecutors.ASYNC_CACHE_MANAGER_EXECUTOR, mWorkerProcess.getWorker(BlockWorker.class));
```

Comment:
I would prefer to have this part of the `workerProcess`, similar to `ufsManager`

Reply:
fixed

Reply:
tried, can not initialize  `AsyncCacheRequestManager` with `getWorker(BlockWorker.class)` in the constr of `WorkerProcess` --- calling the instance method `getWorker` is tricky and fails almost all integration tests.
Thus I rolled back this version